astronomer/providers/microsoft/azure/hooks/wasb.py

```diff
@@ -113,11 +112,11 @@ def _get_container_client(self, container_name: str) -> ContainerClient:
     async def get_blobs_list_async(
         self,
         container_name: str,
-        prefix: Optional[str] = None,
-        include: Optional[List[str]] = None,
+        prefix: str | None = None,
+        include: list[str] | None = None,
         delimiter: str = "/",
         **kwargs: Any,
-    ) -> List[BlobProperties]:
```

Comment:
I think we should inline this with https://github.com/apache/airflow/pull/32545 changes

Reply:
Sure! I've updated it. Could you please check again? Thanks!